configs/devnet/remote-setup-config.yaml

```diff
@@ -35,5 +35,8 @@ devnetBorUsers:
 devnetHeimdallUsers:
   - ubuntu
   - ubuntu
+devnetBorFlags:
```

Comment:
Don't we need to include this also in `configs/devnet/docker-setup-config.yaml` ?

Reply:
Actually, I added it and then later removed it because I saw the following line
```
# Users of hosts where bor will run. Only effective when devnetType is `remote`.
```
[here](https://github.com/maticnetwork/matic-cli/blob/master/configs/README.md).

Should I add it again?

Reply:
I think we need, otherwise docker-setup will only work with cli options.
The comment you mentioned is because with docker setup there will only be one host (ubuntu) running 3 nodes in different containers, but each container running bor would need to be initialised with flags/cli too.

Reply:
Added [here](https://github.com/maticnetwork/matic-cli/pull/215/commits/43a6d60e655eed24ba8391965300d1f64b24350b). Thanks!